mocks/client_mock.go

```diff
@@ -864,6 +864,21 @@ func (mr *MockClientMockRecorder) GetUsers(arg0, arg1, arg2 interface{}) *gomock
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetUsers", reflect.TypeOf((*MockClient)(nil).GetUsers), arg0, arg1, arg2)
 }
 
+// GetUsersInTeam mocks base method
+func (m *MockClient) GetUsersInTeam(arg0 string, arg1, arg2 int, arg3 string) ([]*model.User, *model.Response) {
+	m.ctrl.T.Helper()
+	ret := m.ctrl.Call(m, "GetUsersInTeam", arg0, arg1, arg2)
```

Comment:
> i have added test for the feature but it is failing but not sure why as I have not worked with testing mechanism mattermsot is using so any pointers on how to figure it out are appreciated.

This is what's causing the issue. The call is expecting 4 arguments, but only 3 given.
To fix this you can run `make mocks`, commit and push the changes.

Reply:
Thanks! Fixed it.